content/applications/general/email_communication/faq.rst

```diff
@@ -1,173 +1,225 @@
+- Wait until 11 PM (UTC) for the reset, and click on the retry button. In :ref:`developer mode
+  <developer-mode>`, go to :menuselection:`Settings app --> Technical menu --> Email --> Emails`,
+  then click the :guilabel:`Retry` button next to an unsent email.
 
 .. warning::
-   The daily limit is global to your database and can rise quickly! By default an internal message,
-   a notification, a note, etc. counts as an email in your daily limit if it notifies someone.
+   The daily email limit is comprehensive to the database. By default, an internal message, a
+   notification, a note, etc. counts as an email in the daily limit if it notifies someone via
+   email. This can be mitigated by receiving :ref:`notifications in Odoo
+   <discuss_app/notification_preferences>`, instead of emails.
 
-You can mitigate this by receiving your :ref:`notifications in Odoo <discuss_app/notification_preferences>`
-instead of by emails.
-
-SMTP Error
+SMTP error
 **********
 
-You can find out why an email wasn't transmitted successfully by reviewing the Simple Mail
-Transport Protocol (SMTP) error messages. SMTP is a protocol to describe the email structure
-and transmit it over the Internet, and the error messages generated by email services are helpful
-tools to diagnose and troubleshoot email problems.
-
-No Error
-********
-
-Odoo is not always capable of providing information for the reason it failed. The different
-providers implement a personalized policy of the bounce emails and it is not always possible
-for Odoo to interpret it correctly.
-
-If you have this problem on a recurring basis with the same client or the same domain, please
-do not hesitate to contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
-
-Note: in such case, one of the most common reasons is related to :ref:`SPF <email_communication/spf_compliant>`
-and/or :ref:`DKIM <email_communication/DKIM_compliant>` configuration.
-
-Why is my email sent late?
-**************************
-
-It may happen that you schedule an email campaign but it is not sent on time. We know that
-we use a delayed job to send emails that we consider as not urgent (Newsletters concept
-such as mass mailing, marketing automation, events). The system utility **cron** can be used
-to schedule programs to run automatically at predetermined intervals. We use that policy in order
-to avoid cluttering the mail servers and prioritize the communication.
-
-The emails considered urgent (communication from one person to another one such as
-Sales Orders, Invoices, Purchase Orders, etc.) are sent directly.
+Simple Mail Transport Protocol (SMTP) error messages explain why an email wasn't transmitted
+successfully. :abbr:`SMTP (Simple Mail Transport Protocol)` is a protocol to describe the email
+structure, and transmits data from messages over the Internet. The error messages generated by email
+services are helpful tools to diagnose and troubleshoot email problems.
```

Comment:
These are unique to the service, there are general errors but it is up to each provider to make the error.

Reply:
Great, I think it might be helpful to add that as a quick sentence to the end of this paragraph, but it's already kind of implied by the last sentence, so maybe it's not necessary :)